vdom/core.py

```diff
@@ -179,10 +179,12 @@ def encode_attributes(self):
     def to_dict(self):
         """Converts VDOM object to a dictionary that passes our schema
         """
-        attr_tuple = (self.attributes.items(), {"style": dict(self.style)}.items()) if self.style else (self.attributes.items(),)
+        attributes = self.encode_attributes()
+        if self.style:
+            attributes.update({"style": dict(self.style.items())})
         vdom_dict = {
             'tagName': self.tag_name,
-            'attributes': dict(itertools.chain.from_iterable(attr_tuple))
```

Comment:
We could use the new object spread syntax if we're not worried about supporting Python 2:

```py
'attributes': {**self.encode_attributes(), **{"style": dict(self.style.items())}} if self.style else self.encode_attributes()
```

Reply:
Yes that's exactly why I was using `itertools.chain.from_iterable` was I wanted to use Python3 dict deconstruction… but we need to support both for now.